evaluation.py

```diff
@@ -819,7 +823,9 @@ def sim_true_and_inferred_ancestors(args):
     sample_data = generate_samples(ts, args.error)
 
     inferred_anc = tsinfer.generate_ancestors(sample_data, engine=args.engine)
-    true_anc = tsinfer.AncestorData(sample_data)
+    true_anc = tsinfer.AncestorData(
+        sample_data.sites_position, sample_data.sequence_length
```

Comment:
Trivial, but if you want to shorten `sample_data` to `sd` (which we do sometimes anyway in the tests), it will make all this fit on a single line. There are a whole set of other examples like this too.